--- /usr/lib/python3/dist-packages/neutron/plugins/ml2/drivers/linuxbridge/agent/arp_protect.py.orig	2020-11-02 21:03:14.178809565 -0700
+++ /usr/lib/python3/dist-packages/neutron/plugins/ml2/drivers/linuxbridge/agent/arp_protect.py	2020-11-02 21:24:23.517602651 -0700
@@ -90,6 +90,7 @@
         if chain_exists(chain_name(vif), current_rules):
             LOG.info("deleting chain %s table %s",
                      chain_name(vif), table=table)
+            ebtables(['-F', chain_name(vif)], table=table)
             ebtables(['-X', chain_name(vif)], table=table)
     _delete_mac_spoofing_protection(vifs, current_rules, table=table,
                                     chain=chain)
@@ -216,6 +217,7 @@
     for vif in vifs:
         chain = _mac_chain_name(vif)
         if chain_exists(chain, current_rules):
+            ebtables(['-F', chain], table=table)
             ebtables(['-X', chain], table=table)
 
 
