--- /usr/lib/python3/dist-packages/oslo_service/__init__.py.orig	2020-04-07 04:07:58.000000000 -0600
+++ /usr/lib/python3/dist-packages/oslo_service/__init__.py	2020-11-02 18:14:47.355543590 -0700
@@ -13,13 +13,10 @@
 import os
 
 import eventlet.patcher
-from oslo_log import log as logging
 
 time = eventlet.patcher.original('time')
 
 
-LOG = logging.getLogger(__name__)
-
 # TODO(bnemec): When we have a minimum dependency on a version of eventlet
 # that uses monotonic by default, remove this monkey patching.
 if hasattr(time, 'monotonic'):
@@ -43,3 +40,6 @@
 
 
 os.environ['EVENTLET_HUB'] = 'oslo_service:service_hub'
+
+from oslo_log import log as logging
+LOG = logging.getLogger(__name__)
